ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -101,6 +102,7 @@ public GrpcLogAppender(RaftServer.Division server, LeaderState leaderState, Foll
 
     lock = new AutoCloseableReadWriteLock(this);
     caller = LOG.isTraceEnabled()? JavaUtils.getCallerStackTraceElement(): null;
+    errorWaitStrategy = new ErrorWaitStrategy("5,1000,10,5000");
```

Comment:
The wait times should depend on `getServer().properties().minRpcTimeoutMs()`.

Reply:
Updated...